tests/models/test_models_unet_3d_condition.py

```diff
@@ -193,23 +195,226 @@ def test_model_attention_slicing(self):
             output = model(**inputs_dict)
         assert output is not None
 
-    # (`attn_processors`) needs to be implemented in this model for this test.
-    # def test_lora_processors(self):
+    def test_lora_processors(self):
+        init_dict, inputs_dict = self.prepare_init_args_and_inputs_for_common()
+
+        init_dict["attention_head_dim"] = 8
+
+        model = self.model_class(**init_dict)
+        model.to(torch_device)
+
+        with torch.no_grad():
+            sample1 = model(**inputs_dict).sample
+
+        lora_attn_procs = {}
+        for name in model.attn_processors.keys():
+            has_cross_attention = name.endswith("attn2.processor") and not (
```

Comment:
@Pie31415 I went into your PR here to help a bit, hope that's fine :-) This test passes now I think. 

Feel free to factor this logic here out into a new `create_lora_layers` function for this PR and then it can be re-used in the PRs below to make them pass I think :-)